automate

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
This is nice but it only supports linux machines right? should be mentioned in my opinion